.circleci/config.yml

```diff
@@ -6,12 +6,6 @@ defaults: &defaults
     - image: circleci/node:8
   working_directory: ~/snyk-docker-plugin
 
-release_defaults: &release_defaults
-  resource_class: small
-  docker:
-    - image: node:10
-  working_directory: ~/snyk-docker-plugin
-
```

Comment:
Why remove the whole config but not put the same properties in the commands below? E.g. is the default resource_class: small and the working_directory ~/snyk-docker-plugin